dsgrid/config/dataset_config.py

```diff
@@ -31,53 +31,59 @@
 logger = logging.getLogger(__name__)
 
 
-def check_load_data_filename(path: Path):
+def check_load_data_filename(path: str):
     """Return the load_data filename in path. Supports Parquet and CSV.
 
     Parameters
     ----------
-    path : Path
+    path : str
 
     Returns
     -------
-    Path
+    str
 
     Raises
     ------
     ValueError
         Raised if no supported load data filename exists.
 
     """
+    if path.startswith("s3://"):
+        return path + "/load_data.parquet"
+
     for allowed_name in ALLOWED_LOAD_DATA_FILENAMES:
-        filename = path / allowed_name
+        filename = Path(path) / allowed_name
         if filename.exists():
-            return filename
+            return str(filename)
 
     # Use ValueError because this gets called in Pydantic model validation.
     raise ValueError(f"no load_data file exists in {path}")
 
 
-def check_load_data_lookup_filename(path: Path):
+def check_load_data_lookup_filename(path: str):
     """Return the load_data_lookup filename in path. Supports Parquet, CSV, and JSON.
 
     Parameters
     ----------
-    path : Path
+    path : str
 
     Returns
     -------
-    Path
+    str
 
     Raises
     ------
     ValueError
         Raised if no supported load data lookup filename exists.
 
     """
+    if path.startswith("s3://"):
+        return path + "/load_data_lookup.parquet"
```

Comment:
Why wouldn't this also use ALLOWED_LOAD_DATA_LOOKUP_FILENAMES? 

I also recall us creating a filesystem interface that had both local and cloud instantiations--should we be using that here instead of introducing if-thens?